assets/css/login.css

```
@@ -0,0 +1,70 @@
+#add-post {
+    display: none;
+}
+
+.header{
+    position: fixed;
+    top: 0;
+    width: 100%;
+}
+
+.footer{
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+}
+
+.login-container {
+    width: auto;
+    height: 100%;
+    display: flex;
+    align-items: center;
+}
+
+.login-content {
+    text-align: center;
+    border: 2px solid #ccc;
+    padding: 15px 80px 5px 80px;
+    font-size: 1.25em;
+    background-color: #e2e2e2;
+    width: 100%;
+}
+
+.login-content p {
+    padding-bottom: 15px;
+}
+
+.form-group {
+    margin-bottom: 15px;
+}
+
+.login-content h2 {
+    margin-bottom: 15px;
+    color: #ba181b;   
+}
+
+.button {
+    font-size: 1em;
+    padding: 4px;
+    width: 100%;
+}
```

```suggestion
.button {
    font-size: 1em;
    padding: 4px;
    width: 100%;
    border-radius: 999px;
    border: 2px solid #fff;
    transition: 0.5s, border-width ease;
}
```
Usually buttons look better when they are rounded. The easiest way to achieve that is to define the border radius as a weirdly large pixel size.

Also I added the white border and the transition to make the hover effect work